base/wagtail_hooks.py

```diff
@@ -0,0 +1,25 @@
+from wagtail.snippets.models import register_snippet
+from wagtail.snippets.views.snippets import SnippetViewSet, SnippetViewSetGroup
+
+from base.models import FooterText
+from blog.models import Author
+
+
+class FooterTextViewSet(SnippetViewSet):
+    model = FooterText
+    icon = "pilcrow"
+    search_fields = ("body",)
+
+
+class AuthorViewSet(SnippetViewSet):
+    model = Author
+    icon = "user"
+
+
+class SiteDataSnippetViewSetGroup(SnippetViewSetGroup):
+    menu_label = "Site Data"
+    menu_icon = "info-circle"  # change as required
+    menu_order = 300  # will put in 4th place (000 being 1st, 100 2nd)
+    items = (FooterTextViewSet, AuthorViewSet)
+
+register_snippet(SiteDataSnippetViewSetGroup)
```

Comment:
Genuine question: do these go in wagtail_hooks.py, or better in admin.py?

Reply:
We’ve decided to scrap this for now – technically matching what the bakerydemo does but it feels way too complex for the kind of site we’re building currently.